app/components/trainees/filters/view.html.erb

```diff
@@ -11,7 +11,7 @@
 
       <div class="moj-filter__content">
 
-        <% if !active_filters.empty? %>
+        <% if filters && filters.any? %>
```

Comment:
Should this be `has_filters?` or something like that in the view.rb?

Reply:
Good point! This made me realise it was clunky to sometimes return `{}` and sometimes `nil` from the `TraineeFilter` model. I've modified it to return `nil` in all applicable cases, so we can simply check for `filters`.